test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -14647,6 +14647,11 @@ def __init__(self, job, cmd, host):
         self.cmd = cmd
         self.jobid = None
         self.hostname = host
+        self.runas_user = ""
```

Comment:
set client = "" here
and assign it a value in the if shasta block.

we will not need another if shasta below.